include/ninshiki_cpp/utils/circles.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2021 ICHIRO ITS
```

Comment:
Isn't this done in branch refactor ninshiki ? Maybe remove all changes that are done in refactor ninshiki branch before merging this branch later